the-blue-alliance-ios/View Controllers/Match/MatchInfoViewController.swift

```diff
@@ -117,7 +117,7 @@ class MatchInfoViewController: TBAViewController, Observable {
 
     func styleInterface() {
         scrollView.addSubview(matchStackView)
-        matchStackView.autoSetDimension(.height, toSize: 90)
+        matchStackView.autoSetDimension(.height, toSize: matchSummaryView.calculatedHeight)
```

Comment:
This is tricky... I think the right way to go about doing this is actually inside of the `MatchSummaryView`, and use constraints to set a minimum size of 44px height on the buttons we generate, and then leave the labels as having whatever height they already have. That way we'll be able to show big buttons where we want buttons, and small text where we want text, and we don't have to mess with the stack view much.

This constraint can probably go away though I hope, if the system has enough information to size itself based on the button heights. Also we'll probably want to change that ` infoStackView.autoMatch(.height, to: .height, of: matchStackView, withMultiplier: (1.0/3.0))` line below. We should just go ahead and set it to be 30px.